src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
```

Comment:
Should this be styled to look more like a button to stay on-brand (blue w/ white text)?